.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
       - name: invoke build
         if: ${{ github.ref == 'refs/heads/main' }}
-        uses: benc-uk/workflow-dispatch@v1.1.0
+        uses: benc-uk/workflow-dispatch@v1
```

Comment:
1.1.0 doesn't appear to be usable anymore - unpinned so that we can use the newest minor 1.x version